jsk_pcl_ros/src/linemod_nodelet.cpp

```diff
@@ -553,10 +553,11 @@ namespace jsk_pcl_ros
     pcl::PCDReader reader;
     reader.read(template_file_ + ".pcd", *template_cloud_);
     const std::string pose_yaml = template_file_ + "_poses.yaml";
+   YAML::Node doc;
+#ifdef USE_OLD_YAML
     std::ifstream pose_fin;
     pose_fin.open(pose_yaml.c_str(), std::ifstream::in);
     YAML::Parser parser(pose_fin);
-    YAML::Node doc;
     while (parser.GetNextDocument(doc)) {
```

Comment:
cc:@garaemon

it is not necessary at this moment, may be we'll have to fix this 6 month later, 
this is wrong, it deletes necessary code for new yaml, i think this assumes you have several `template_poses` entry in yaml is it correct?
```
template_poses:
   hoge hogehoge
template_poses:
  hogehog hogheo
```
is it correct? or you only have one `template_poses`? if so, whty you need `parser.GetNextDocument(doc)`? changing to 
```
parser.GetNextDocument(doc)
const YAML::Node& template_pose_yaml = doc["template_poses"];
for (size_t i = 0; i < template_pose_yaml.size(); i++) {
 ...
}
```
will work?